src/functions/getProduct/getProduct.ts

```diff
@@ -0,0 +1,21 @@
+import { formatJSONResponse } from "../../libs/api-gateway";
+import { middyfy } from "../../libs/lambda";
+import { product } from "./product.mock";
+
+export const getProduct = async (event) => {
```

Comment:
Since you're in typescript you could use special types for API gateway lambda headers